Coroutines/declarators.tex

```diff
@@ -112,7 +112,7 @@
      co_await @$p$@.initial_suspend(); // initial suspend point
      try { @\textit{F}@ } catch(...) { @\textit{p}@.unhandled_exception(); }
   @\textit{final_suspend}@:
-     co_await @$p$@.final_suspend(); // final suspend point
+     // final suspend point
```

Comment:
Does this still imply that the promise object and frame would be destructed if an exception was thrown from call to final_suspend()? Do we need to add some more wording to clarify that this is not the case?

Reply:
Fixed by: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p1356r0.html